### PR TITLE
TuiEditor namespace error and Get Tui-viewer

### PR DESCRIPTION
**Pull request by dc7303**
First of all thank you for your wonderful project. 
And before you review this, please understand my lack of English.

I am using ngx-tui-editor in my project. But there were some issues in applying to the project.
1. Can't use namespace ‘TuiEditor’ as a type     #7 
2. Can't load Tui-viewer

I am modifying the code that I have installed and applying it to my project. It seems to be convenient.

### CODE REVIEW

**Comment by tylernhoward** [comment]
Thanks for the PR!
To be clear, this should also resolve #7 ?

File: src/tui-editor.service.ts [hunk 1]
```diff
@@ -5,12 +5,21 @@ export class TuiService {
   editor: any;
   constructor() { }
   createEditor(options: object): any {
-    this.editor = new TuiEditor(Object.assign({
-      el: document.querySelector('.ngx-tui-editor'),
-      initialEditType: 'markdown',
-      previewStyle: 'vertical',
-      height: '300px'
-    }, options));
+    if (options.viewer) {
```
Comment (tylernhoward) [inline src/tui-editor.service.ts]: When you say: "Can't load Tui-viewer" - What error are you seeing exactly that this is addressing?

Comment (dc7303) [reply src/tui-editor.service.ts]: Thanks for the reply!
Changed to any type to solve # 7.  When I used it, I have to download the package and modify it with any type.

Can not load The Tui-viewer does not tell you the error message.
The Tui-editor documentation provided by nhn uses the 'static factory method' when using the viewer. I thought I should be able to use the viewer by adding the 'viewer: true' value to the option object while using the ngx-tui-editor.

I told you before, but my English skills are immature. If I misunderstood the intent of the question, can you explain it in more detail?

Comment (tylernhoward) [reply src/tui-editor.service.ts]: Your English is great! This answers my question. Looks good to me, thanks for the contribution!

**Comment by tylernhoward** [comment]
Closes #7

**Pull request merged**